### ISSUE TITLE
Exception thrown if selected USB device is not mounted

### ISSUE THREAD

**Issue by bagage** [mbusb/multibootusb#83]
### Issue

Running master, when I select an ISO file and validate it, the following exception is thrown:

```
% sudo ./multibootusb
Starting multibootusb GUI...
Using pyudev for detecting USB drives...
UBS disk is not mounted and can't update list widget...
Cleaning old multibootusb directory...
Extracting syslinux to multibootusb directory...
Host OS is 64 bit...
Extracting syslinux 64 bit...
Extracting syslinux modules to multibootusb directory...
Traceback (most recent call last):
  File "/home/gautier/multibootusb/scripts/mbusb_gui.py", line 183, in browse_iso
    per_availability = persistence.persistence_distro(config.distro, config.usb_disk, config.iso_link)[0]
  File "/home/gautier/multibootusb/scripts/persistence.py", line 34, in persistence_distro
    if usb_free_size > fat_max_size:
TypeError: unorderable types: str() > int()
```

### Workaround

I guess this is somewhat related to the USB key not properly formated (it was already filled with a previous ISO installed via `dd`). To fix this issue, I have to fast-format the key to FAT, unplug it, replug it and restart the application.

**Comment by mbusb** [comment]
The first and main requirement for using multibootusb is that you should have fat formatted USB drive.

**Comment by bagage** [comment]
Update: the issue happens when the selected device is actually not mounted. Either the application should mount it before going further, or stop immediately?

**Comment by mbusb** [comment]
Fixed in the latest release (8.5.0).
